pkg/report/decompile.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/syzkaller/pkg/osutil"
+	"github.com/google/syzkaller/sys/targets"
+)
+
+type DecompilerFlag string
+
+// Extra flags that control the flow of decompilation.
+const (
+	FlagForceArmThumbMode DecompilerFlag = "force_arm_thumb"
+)
+
+const objdumpCallTimeout = 10 * time.Second
+
+type DecompiledOpcode struct {
+	Offset  int
+	Len     int
+	Command string
```

Comment:
Is Len and Command used anywhere? If not, please remove.
Generally adding code "just in case" is not a good idea. It's hard to predict what will be needed and in what form. Dead code also frequently breaks, so even if it's needed in future, it's not necessary it's ready to use.